### DISCUSSION TITLE
posts/fuzzing-heartbleed-to-xz/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#32] [Announcements]
# posts/fuzzing-heartbleed-to-xz/

Fuzzing漫谈：Heartbleed，AI与xz投毒事件

https://blog.wohin.me/posts/fuzzing-heartbleed-to-xz/

**Comment by snowcgj** [comment]
在外高群里看到阮博男学长这个名字，高中期间只听闻过席硕玉席老师提起过阮学长的优秀，没想到网上一搜竟真真正正搜到了阮学长的博客。 首先是github上对开源社区的贡献，然后是简介上baset paper的作者，本以为这就已经很天才了，已经一直在内心感慨了，没想到打开这个随笔，还这么有文采，天啊，真的好优秀的学长，哈哈哈，高中学长。忍不住感慨，真的好厉害。 最后，祝阮学长在新加坡求学顺利，成果多多，又多了一个榜样。

**Reply by brant-ruan** [reply]
哈哈，谢谢你～外高人到哪都不会差，一起加油 :-)

**Reply by snowcgj** [reply]
好！
